.buildkite-platform.json

```diff
@@ -1,6 +1,6 @@
 {
     "chef_foundation": "3.2.16",
-    "omnibus_toolchain": "3.0.36",
-    "ruby_version": "3.1.6",
-    "bundle_version": "2.3.7"
+    "omnibus_toolchain": "3.0.39",
```

Comment:
minor note - if we nuke omnibus first in it's own PR, this PR (and a few others) become cleaner. Not blocking on that just a nitpicky note.

Reply:
@tpowell-progress reset this line to the original value

Reply:
@jaymzh Oh, the "omnibus_toolchain" is still the name of the underlying build platform, despite not being strictly omnibus.